.tito/tito.props

```diff
@@ -4,3 +4,4 @@ tagger = tito.tagger.VersionTagger
 changelog_do_not_remove_cherrypick = 0
 changelog_format = %s
 fetch_sources = True
+test_version_suffix = .test
```

Comment:
Nit: would `.git` be better?

Reply:
The difference would be:

```diff
/tmp/tito/copr-backend-2.1.test-1.git.38.3719a4f.fc41.src.rpm
/tmp/tito/copr-backend-2.1.git-1.git.38.3719a4f.fc41.src.rpm
```

Nobrainer, just note there's the double-`git` part `git-1.git`.